cmake/scripts/common/AddonHelpers.cmake

```diff
@@ -120,16 +120,20 @@ macro (build_addon target prefix libs)
             foreach(src_file ${USED_SOURCES})
               file(STRINGS ${src_file} BIN_ADDON_SRC_PARTS)
               foreach(loop_var ${BIN_ADDON_SRC_PARTS})
+                string(REGEX REPLACE ".*#include" "#include" loop_var "${loop_var}")
```

Comment:
This `REGEX REPLACE` is error prone, as it would match `#includes` which have other characters the spaces at the begin. Also it doesn't take care about spaces after the `#` or using `#import` (Objective-C++).

Reply:
Have added a space handling after `#` and also tanslate of `#import`